vpr/src/place/place.cpp

```diff
@@ -49,6 +51,15 @@
 
 #include "placer_breakpoint.h"
 
+//define the reward function factor constant
```

Comment:
What is the reward function factor constant?

Reply:
It is a factor that controls the relative weights of timing and bb cost. I have added a comment to explain it more and also changed its name to match its functionality